extensions/amp-apester-media/0.1/test/validator-amp-apester-media.out

```diff
@@ -0,0 +1 @@
+PASS
```

Comment:
I think it might be nice to add a couple invalid examples as well.

<!-- Invalid: neither playlist nor individual media id provided -->
...
<!-- Invalid: both playlist and individual media id provided -->
...

Basically you'd put these two examples into the validator-amp-apester-media.html file and then record the expected error messages in this file. You can just run the build.py script to let the test fail and then take the error messages from there and put them here to make stuff pass.

It  will look similar to https://github.com/ampproject/amphtml/blob/master/extensions/amp-youtube/0.1/test/validator-amp-youtube.out (but the error messages will be different).